plugins/processors/awsappsignals/internal/attributes/attributes.go

```diff
@@ -5,38 +5,20 @@ package attributes
 
 const (
 	// aws attributes
-	AWSLocalService        = "aws.local.service"
-	AWSLocalOperation      = "aws.local.operation"
-	AWSRemoteService       = "aws.remote.service"
-	AWSRemoteOperation     = "aws.remote.operation"
-	AWSRemoteTarget        = "aws.remote.target"
-	AWSHostedInEnvironment = "aws.hostedin.environment"
+	AWSLocalService             = "aws.local.service"
+	AWSLocalOperation           = "aws.local.operation"
+	AWSRemoteService            = "aws.remote.service"
+	AWSRemoteOperation          = "aws.remote.operation"
+	AWSRemoteTarget             = "aws.remote.target"
+	AWSRemoteResourceIdentifier = "aws.remote.resource.identifier"
+	AWSRemoteResourceType       = "aws.remote.resource.type"
+	AWSHostedInEnvironment      = "aws.hostedin.environment"
```

Comment:
we are still keeping this attribute name?

Reply:
No. But since the attribute is generated SDK side, we want to make sure it won't break the new metric generation in case customers upgrade CWA first, SDK second. Thus `aws.hostedin.environment` is still referenced when generating `Environment` dimension.